content/en/docs/Reference/catalog-templates.md

```diff
@@ -254,6 +254,9 @@ The `olm.semver` [cue](https://cuelang.org/docs/references/spec/) schema is:
   // optional flag to control generating major-version channels, defaults to _false_ if unspecified
   GenerateMajorChannels?: bool
 
+  // optional preference for major- or minor-version channels, when both are generated and identical in stability and version
+  DefaultChannelTypePreference?: string
```

Comment:
Is there a requirement that this is within a specific set of values or can this be any string value?

Reply:
We enforce it in the code to either 'minor' or 'major': https://github.com/operator-framework/operator-registry/blob/master/alpha/template/semver/semver.go#LL95-L112